.circleci/config.yml

```diff
@@ -637,7 +609,26 @@ jobs:
             - yarn-deps-lock-{{ checksum "ui/yarn.lock" }}
       - run: sudo apt-get update && sudo apt-get install netcat-openbsd
       - run:
-          command: ./bin/linux/influxd --store=memory --e2e-testing=true
+          name: Install Dependencies
+          command: |
+            set +e
```

Comment:
Why is `e` being turned off here?

Reply:
Beyond that I just copied it verbatim from [where it was before](https://github.com/influxdata/influxdb/blob/e28377f05cc92cadee11029e4b9046aaedfd32b6/.circleci/config.yml#L297) I couldn't say to tell you truth. Doesn't seem like it would be strictly necessary but I also didn't try without. Soon this whole `e2e` job will be revamped and replaced with the `monitor-ci` pipeline so it should be out the picture then at least!